src/libcollectdclient/network_parse.c

```diff
@@ -535,7 +535,7 @@ static int network_parse(void *data, size_t data_size, lcc_security_level_t sl,
 
     if ((sz < 5) || (((size_t)sz - 4) > b->len)) {
       DEBUG("lcc_network_parse(): invalid 'sz' field: sz = %" PRIu16
-            ", b->len = %" PRIsz "\n",
+            ", b->len = %" PRIu64 "\n",
```

Comment:
Is this the only file affected by this issue? Seems like it would apply to any use of the `PRIsz` format with `size_t`… Why not change the [definition of `PRIsz` in `globals.h`](https://github.com/Stackdriver/collectd/blob/campbellsean-windows-4/src/daemon/globals.h#L35)? Also, this should be fixed more comprehensively by 973ffb1a767cd1cc21bc9a344d4cb6dbe96883cc (on the [igorpeshansky-windows-4](https://github.com/Stackdriver/collectd/tree/igorpeshansky-windows-4) branch).